readme.md

```diff
@@ -456,6 +458,14 @@ test.cb(t => {
 });
 ```
 
+### Optional TAP output
+
+AVA can generate TAP output via `--tap` option for use with any 3rd-party reporters.
+
+```bash
```

Comment:
Was using it, because sometimes it highlights executable name and symbols like `|`. Will remove it, no probs ;)